helm/cluster-azure/values.schema.json

```
@@ -197,8 +197,29 @@
                     "title": "Identity",
                     "type": "object"
                 },
+                "image": {
+                    "properties": {
+                        "gallery": {
+                            "default": "gsCAPITest1-5cb24dcf-a2d0-4aba-820f-b52ca78f96e6",
+                            "title": "Name of the Community gallery hosting the image",
+                            "type": "string"
```

I'd recommend a shorter title, with a description for verbose info.

(Caution: the white space probably needs normalization)

```suggestion
                            "default": "gsCAPITest1-5cb24dcf-a2d0-4aba-820f-b52ca78f96e6",
                            "description": "Name of the community gallery hosting the image",
                            "title": "Gallery",
                            "type": "string"
```